urbanopt/geojson-modelica-translator
# Use `filNam` parameter for TEASER loads and add within parsing to PackageParser

#### Any background context you want to provide?
It appears to be customary to place filename variables as parameters at the top of the .mo files. For the TEASER workflow, we have been updating the path to the load files in the CombiTimeTable object directly. In addition, there were no methods for parsing out the within block of a package.mo file.

#### What does this PR accomplish?
* Moves the filename of the TEASER zones into a parameters at the top of the files
* Adds get and update methods to the package.mo parser
* Removes the <package_name> object when saving to the `DistrictEnergySystem.mo` file. In Dymola (and presumably in OMC and others), this is not needed and made downstream model manipulations a bit more difficult. 

#### How should this be manually tested?
* CI should suffice to catch the majority of the issues.
* Run test_teaser and then manually run the resulting the `teaster_single` model in Dymola.

#### What are the relevant tickets?
n/a

#### Screenshots (if appropriate)
